tests/queries/0_stateless/03237_create_or_replace_view_atomically_with_atomic_engine.sh

```diff
@@ -18,7 +19,7 @@ export -f create_or_replace_view_thread;
 function select_view_thread
 {
     for _ in {1..20}; do
-        $CLICKHOUSE_CLIENT --query "SELECT * FROM ${CLICKHOUSE_DATABASE}_db.test_view" > /dev/null
+        $CLICKHOUSE_CLIENT --query "SELECT * FROM ${CLICKHOUSE_DATABASE}_db.test_view FORMAT NULL"  | grep -v -P 'Code: (60|741)'
```

Comment:
BTW this does not fix it at all, since `grep` only stdout

Reply:
@azat, You are totally right, but better to fix it immediately instead of giving me 2.5 months of suffering? https://github.com/ClickHouse/ClickHouse/pull/80412